peertalk/PTChannel.m

```diff
@@ -214,7 +214,20 @@ - (void)connectToPort:(int)port overUSBHub:(PTUSBHub*)usbHub deviceID:(NSNumber*
 }
 
 
-- (void)connectToPort:(in_port_t)port IPv4Address:(in_addr_t)ipv4Address callback:(void(^)(NSError *error, PTAddress *address))callback {
+- (void)connectToPort:(uint16_t)port IPv4Address:(NSString*)address callback:(void(^)(NSError *error, PTAddress *address))callback {
```

Comment:
This should be a separate implementation. Using `in_addr_t` is often practically more useful, so that method mustn't go away. Also, a method accepting a text-form address should also handle DNS names, not just IP addresses. This means blocking I/O, which needs to be clearly stated to the user (or implemented in an async manner.)

If all one needs is to convert a text IP address to a numeric IP address at runtime, a one-liner using `inet_aton` should be enough. Let's keep the API simple :)